From 54c795c8a3c7356294007b5a4eed1dd47ed6411d Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 11 May 2019 19:19:27 +0800
Subject: [PATCH] fix shebang for target

...
|ERROR: groff-1.22.4-r0 do_package_qa: QA Issue: /usr/bin/gdiffmk contained in
package groff requires tmp-glibc/hosttools/bash, but no providers found in
RDEPENDS_groff? [file-rdeps]
...

Upstream-Status: Inappropriate [oe-core specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 contrib/gdiffmk/gdiffmk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/gdiffmk/gdiffmk.sh b/contrib/gdiffmk/gdiffmk.sh
index 5ce931e..10f2300 100644
--- a/contrib/gdiffmk/gdiffmk.sh
+++ b/contrib/gdiffmk/gdiffmk.sh
@@ -1,4 +1,4 @@
-#!@BASH_PROG@
+#!/bin/sh
 # Copyright (C) 2004-2018 Free Software Foundation, Inc.
 # Written by Mike Bianchi <MBianchi@Foveal.com <mailto:MBianchi@Foveal.com>>
 # Thanks to Peter Bray for debugging.
-- 
2.7.4

